### DISCUSSION TITLE
How do you UPDATE a row's geometry?

### DISCUSSION THREAD

**Discussion by sentientnebula** [realiii/fudgeo#80] [Q&A] [answered]
SQL UPDATE Statements work on all columns except the geometry (SHAPE) column.  What is the proper syntax to update (replace) the value of a row's geometry column?

Also, Is there a way to name the geometry column something other than SHAPE when creating a new Feature Class?

**Comment by jlhumber** [comment, marked as answer]
Take a look at this [commit](https://github.com/realiii/fudgeo/commit/aaae72bfb552fbe0a73ba2add07dea9b8d40246a) -- I just now added this to ensure it was working as expected.

As for creating a Feature Class with a geometry column other than `SHAPE`, it is not supported at the moment.  I have added #82 to address this.

**Reply by sentientnebula** [reply]
Wow!  Thanks for the quick response, and lightning fast new feature implementation.  ++Karma
